phtc/main/tests/test_views.py

```diff
@@ -11,7 +11,7 @@ def setUp(self):
 
     def test_index(self):
         result = self.c.get("/")
-        self.assertEqual(result.status_code, 200)
+        self.assertEqual(result.status_code, 301)
```

Comment:
I'd recommend against using 301 redirects within the app. A 301 is permanent and browsers cache them forever, so they can really never be undone.